src/ui/units/collections/components/CollectionContent/CollectionContent.tsx

```diff
@@ -152,16 +209,255 @@ export const CollectionContent = React.memo<Props>(
+                        );
+                    },
+                });
+            }
+
+            if (item.permissions.move) {
+                actions.push({
+                    text: i18n('action_move'),
+                    action: () => {
+                        dispatch(
+                            openDialog({
+                                id: DIALOG_MOVE_COLLECTION,
+                                props: {
+                                    open: true,
+                                    collectionId: item.collectionId,
+                                    collectionTitle: item.title,
+                                    initialParentId: item.parentId,
+                                    onApply: refreshContent,
+                                    onClose: handeCloseMoveDialog,
+                                },
+                            }),
+                        );
+                    },
+                });
+            }
+
+            if (collectionsAccessEnabled && item.permissions.listAccessBindings) {
+                actions.push({
+                    text: i18n('action_access'),
+                    action: () => {
+                        setDialogState(DialogState.EditCollectionAccess);
+                        setDialogEntity(item);
+                    },
+                });
+            }
+
+            if (item.permissions.delete) {
+                actions.push({
+                    text: i18n('action_delete'),
+                    action: () => {
+                        setDialogState(DialogState.DeleteCollection);
+                        setDialogEntity(item);
+                    },
+                    theme: 'danger',
+                });
+            }
+
+            return actions;
+        };
+
         return (
             <React.Fragment>
-                <CollectionContentTable
-                    contentItems={contentItems}
-                    filters={filters}
-                    setFilters={setFilters}
-                    refreshContent={refreshContent}
-                />
+                {collectionPageViewMode === CollectionPageViewMode.Grid ? (
+                    <CollectionContentGrid
+                        contentItems={contentItems}
+                        filters={filters}
+                        setFilters={setFilters}
+                        getWorkbookActions={getWorkbookActions}
+                        getCollectionActions={getCollectionActions}
+                    />
+                ) : (
+                    <CollectionContentTable
+                        contentItems={contentItems}
+                        filters={filters}
+                        setFilters={setFilters}
+                        getWorkbookActions={getWorkbookActions}
+                        getCollectionActions={getCollectionActions}
+                    />
+                )}
+
                 {isContentLoading && <SmartLoader size="m" showAfter={0} />}
```

Comment:
The gap between loader and content is too small:
<img width="699" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/29424746/0d3867c5-914d-4d2d-9382-250410fd7d7c">